### PR TITLE
Fix #18442: About window background is clickable

### LINKED ISSUE #18442: About window is clickable.
### Operating System

Windows 10, 64-bit

### OpenRCT2 build

OpenRCT2, v0.4.2 (8ceea45) provided by GitHub

### Describe the issue

The about window is clickable for some reason.

### Area(s) with issue?

_No response_

### Steps to reproduce

1. Open the about menu

Screenshot2 2022-10-25

2. Click on certain areas

Screenshot3 2022-10-25
Screenshot4 2022-10-25

### Attachments

_No response_

### PR DESCRIPTION
**Pull request by tupaschoal**
Fix #18442

### CODE REVIEW

File: src/openrct2-ui/windows/About.cpp [hunk 1]
```diff
@@ -57,7 +57,7 @@ enum WindowAboutWidgetIdx {
 
 #define WIDGETS_MAIN \
     WINDOW_SHIM(WINDOW_TITLE, WW, WH), \
-    MakeWidget     ({ 0, TABHEIGHT}, {WW, WH - TABHEIGHT}, WindowWidgetType::ImgBtn, WindowColour::Secondary               ), /* page background */       \
+    MakeWidget     ({ 0, TABHEIGHT}, {WW, WH - TABHEIGHT}, WindowWidgetType::Frame, WindowColour::Secondary               ), /* page background */       \
```
Comment (duncanspumpkin) [inline src/openrct2-ui/windows/About.cpp:60]: Can you fix the alignment

Comment (tupaschoal) [inline src/openrct2-ui/windows/About.cpp:60]:
```suggestion
    MakeWidget     ({ 0, TABHEIGHT}, {WW, WH - TABHEIGHT}, WindowWidgetType::Frame,  WindowColour::Secondary               ), /* page background */       \
```

**Comment by rik-smeets** [comment]
Could you rebase this?

**Comment by tupaschoal** [comment]
Done @rik-smeets

**Review by rik-smeets — APPROVED** [review]

**Pull request merged**